src/psqlgraph/base.py

```diff
@@ -7,7 +8,7 @@
 from sqlalchemy.sql import expression, schema, sqltypes
 
 from psqlgraph import attributes, voided
-from psqlgraph.util import sanitize, validate
+from psqlgraph.util import sanitize
```

Comment:
This is an optional suggestion and one that you may already have planned for another PR. Since the sanitize import was replaced with a util module import to reference sanitize in the attributes module, it would be nice to do the same here for consistency.

Reply:
Yeah the base module is in my crosshairs for removal as I'm centralizing all of the various base classes of the graph entities.